src/NzbDrone.Core/Notifications/PushBullet/PushBulletProxy.cs

```diff
@@ -151,6 +151,11 @@ private void SendNotification(string title, string message, RestRequest request,
                 request.AddParameter("title", title);
                 request.AddParameter("body", message);
 
+                if (!String.IsNullOrEmpty(settings.SenderId))
```

Comment:
Its a string extension we've added, coming from https://github.com/Sonarr/Sonarr/blob/develop/src/NzbDrone.Common/Extensions/StringExtensions.cs#L76

Include `using NzbDrone.Common.Extensions` at the top if it doesn't automatically suggest it.